pkg/apmintegration/helpers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package apmintegration
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func getGithubOrg(url string) (string, error) {
+	split := strings.Split(url, "/")
+
+	if len(split) < 3 {
+		return "", errors.New("unable to find organization")
+	}
+
+	return split[len(split)-2], nil
+}
+
+func getGithubRepo(url string) (string, error) {
+	base := filepath.Base(url)
```

Comment:
As this is no filesystem path, but an URL path, the `path` package should be preferred instead of filepath.

Reply:
fixed